File: Problems/0500_0599/0501_Find_Model_in_Binary_Seach_Tree/Project_CS/Mylib.cs
public class Mylib
{
    public int[] StringToIntArray(string s)
    {
        if (s.Length <= 0)
            return null;

        string[] flds = s.Split(",");
        int[] nums = new int[flds.Length];

        if (flds.Length <= 0)
            return nums;

        for (int i = 0; i < nums.Length; ++i) {
            nums[i] = int.Parse(flds[i]);
        }

        return nums;
    }

    public string IntArrayToString(int[] nums)
    {
        if (nums.Length <= 0)
            return "";

        string resultStr = nums[0].ToString();

        for (int i = 1; i < resultStr.Length; ++i)
        {
            resultStr += ", " + nums[i].ToString();
        }

        return resultStr;
    }
}

File: Problems/0001_0099/0002_Add_Two_Number/Project_Scala/Mylib.scala
object Mylib {
    def stringToIntArray(flds:String):Array[Int] = {
        var nums_str:Array[String] = flds.split(",")
        var nums:Array[Int] = new Array[Int](nums_str.size)

        for (i <- 0 until nums_str.length) {
            nums(i) = nums_str(i).toInt
        }

        return nums
    }
 
    def intArrayToString(nums:Array[Int]):String = {
        if (nums.size <= 0)
            return ""

        var resultStr:String = nums(0).toString
        for (i <- 1 until nums.length) {
            resultStr += ", " + nums(i).toString
        }

        return resultStr
    }
}
